### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by MikeTheReader** [trulia/hologram#273]
Having good luck setting this up and getting a nice looking style guide, but not seeing a whole lot of activity on this repository. Is this project still maintained?

Thanks much.

**Comment by jdcantrell** [comment]
We still use this at Trulia, but it's not needed a lot of changes so updates have been pretty long coming. My sense is a lot of newer style guides integrate better with more recent web development changes and that many people would prefer a js solution over ruby. So I personally have not been super enthusiastic about updating this project with new features.

**Comment by MikeTheReader** [comment]
I've been looking for a better solution, and have to say I've had a hard time finding something as "plug and play" as this one. Kudos for making the commenting style and default style so intuitive.

You mentioned newer style guides -- are there any more current ones that stand out for you? (I also 
would prefer ones in the JS realm.)

Thanks again.

**Comment by nickmjones** [comment]
@MikeTheReader [Fabricator](https://fbrctr.github.io) is a pretty popular alternative for folks documenting guides/design systems in JS frameworks. I've used it some and it's nice, but nothing so far is as plug-and-play as hologram.

**Comment by leonardofaria** [comment]
What I liked in Hologram is that I can write markdown like a Jenkyll project without lots of configurations. I spent a few hours last Friday looking the repos in the Readme (specially the Acme and Github-like examples) and I got something working fine based on half-dozen of npm scripts. 

In my case, the CSS I am detailing is external (imagine something like Bootstrap) and Hologram with markdown works like a charm. Didn't find something similar to this goal.
